pkg/wait/conditions.go

```diff
@@ -278,6 +278,29 @@ func AreAllPodsInSuccessfulPhase(ctx context.Context, kubeClient *client.Client)
 	}
 }
 
+// AreAllPodsNotCrashLooping checks that all pods within the cluster have fewer than the provided number of restarts
+func AreAllPodsNotCrashLooping(ctx context.Context, kubeClient *client.Client, maxRestartCount int32) WaitCondition {
```

Comment:
AreNoPodsCrashLooping somehow feels simpler to my brain but otherwise lgtm

Reply:
Done!